business.js

```diff
@@ -15,11 +15,24 @@ $.each(pDetails, function(key, value) {
 
 // business show
 $('.details').click(function() {
+    var dDetails = $('.panel-body')
     var id = $(this).attr('id')
     var persnDetl = pDetails.find(x => x.id == id)
     $('.panel-heading > .panel-title').text('Personal Information')
+    dDetails.html('<b>Name : </b>' + ' ' + persnDetl.name + '<br>').append('<b>Address : </b>' + ' ' + persnDetl.address + '<br>').append('<b>Accounts : </b>' + ' ' + persnDetl.accounts + '<br>').append('<b>Job : </b>' + ' ' + persnDetl.job)
```

Comment:
why are you using extra two appends here remove that and append it by using one predicator instead of 3